cheroot/test/test_conn.py

```diff
@@ -1013,16 +1013,11 @@ def test_No_CRLF(test_client, invalid_terminator):
 
 
 class FaultySelectorManager:
-    """Mock class to insert errors in th execution.
-
-    Wraps: :py:class:`!cheroot.connections._SelectorManager`.
```

Comment:
@cyraxjoe FYI look how I've ignored similar references to undocumented objects in the past: https://github.com/cherrypy/cheroot/blob/d2f28f2/docs/conf.py#L95-L96